plugins/janus_videoroom.c

```diff
@@ -5067,6 +5073,23 @@ gboolean wait_for_pipeline_close(forward_media_type media_type,
 	return TRUE;
 }
 
+gboolean is_pipeline_processing_started(forward_media_type media_type,
```

Comment:
start_pipline_processing_flag
TRUE -> success and start pipeline
FALSE -> fail. already running. ignore